<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COutputDialog</class>
 <widget class="QDialog" name="COutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV saving parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>CSV files saving</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Directory for CSV files</string>
          </property>
          <property name="buddy">
           <cstring>editCSVDir</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editCSVDir"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnCSVDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>File name template</string>
          </property>
          <property name="buddy">
           <cstring>editCSVTemplate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editCSVTemplate"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editCSVDir</tabstop>
  <tabstop>btnCSVDir</tabstop>
  <tabstop>editCSVTemplate</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>COutputDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>COutputDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
